### PR TITLE
Save number cell from setUp function

### PR DESCRIPTION
**Pull request by anvacaru**
Fixes: #133

### CODE REVIEW

File: src/tests/integration/test-data/foundry/test/BlockParamsTest.t.sol [hunk 1]
```diff
@@ -36,3 +36,15 @@ contract BlockParamsTest is Test {
         assert(x >= 0);
     }
 }
+
+contract RollTest is Test {
+    function setUp() external {
+        vm.roll(123);
+    }
+
+    // function testFail_run1() external {
```
Comment (palinatolmach) [inline src/tests/integration/test-data/foundry/test/BlockParamsTest.t.sol:45]: Sorry, that came from the contract I was running — we probably don't need it here :)

Comment (anvacaru) [inline src/tests/integration/test-data/foundry/test/BlockParamsTest.t.sol:45]:
```suggestion
```

Comment (anvacaru) [reply src/tests/integration/test-data/foundry/test/BlockParamsTest.t.sol]: Done. I need to update the expected output once more.

**Comment by palinatolmach** [comment]
Created a separate issue for tracking other similar cells as a follow-up to this PR: https://github.com/runtimeverification/kontrol/issues/169.

**Review by palinatolmach — APPROVED** [review]

**Pull request merged**